galeracluster/source/documentation/install-mysql.rst

```diff
@@ -124,12 +124,12 @@ Version 4 of Galera was recently released.  If you'd like to install it, the con
 .. code-block:: linux-config
 
    # Codership Repository (Galera Cluster for MySQL)
-   deb https://galeracluster.com/wsrep_8.0.19-26.3-rc/galera-4/ubuntu bionic main
-   deb https://galeracluster.com/wsrep_8.0.19-26.3-rc/mysql-wsrep-8.0/ubuntu bionic main
+	deb https://releases.galeracluster.com/galera-4/ubuntu focal main
+	deb https://releases.galeracluster.com/mysql-wsrep-8.0/ubuntu focal main
 
```

Comment:
indentation seems to be with spaces, in other parts of this document, but in these new lines, indentation appears to be done by TABs. If this can cause misaligned lines in the final output, please fix to use spaces for indentation here as well

Reply:
Fixed!